src/lib/components/MobileNav.svelte

```diff
@@ -7,22 +7,43 @@
     import { Button, InlineTag, Icon } from '$lib/components/ui';
     import { GithubStats } from '$lib/components/shared';
     import { trackEvent } from '$lib/actions/analytics';
+    import { browser } from '$app/environment';
+    import { onMount } from 'svelte';
 
     export let open = false;
     export let links: NavLink[];
     export let offerButton = false;
 
+    let bannerHeight = 0;
+
     afterNavigate(() => {
         open = false;
         document.body.style.overflow = '';
     });
+
+    function updateBannerHeight() {
+        if (browser) {
+            const banner = document.getElementById('hackathon-banner');
```

Comment:
we can also cache this assuming the banner is cancellable instead of always fetching the element. something like -
```ts
let banner: HtmlElement | null = null;

function getBannerElement() {
  if (banner !== null) return banner;

  banner = document.getElementById('hackathon-banner');
  if (banner) {
    return banner;
  } else {
    return null;
  }
}
```
then just reference `banner` in the `updateBannerHeight` with a null check /guard because the banner won't necessarily be there always!

Not needed if the banner can be removed via some interaction!